src/e2ee.rs

```diff
@@ -47,7 +47,8 @@ pub struct EncryptHelper {
 impl EncryptHelper {
     pub fn new(context: &Context) -> Result<EncryptHelper> {
         let prefer_encrypt =
-            EncryptPreference::from_i32(context.get_config_int(Config::E2eeEnabled)).unwrap();
+            EncryptPreference::from_i32(context.get_config_int(Config::E2eeEnabled))
+                .unwrap_or_default();
```

Comment:
Not propagating up with `?`?

Reply:
i think this would require some further refactoring as `Error` is not part of the result type.